site/docs-md/apis/status-bar/index.md

```diff
@@ -10,6 +10,8 @@ The StatusBar API Provides methods for configuring the style of the Status Bar,
 
 This plugin requires "View controller-based status bar appearance" (`UIViewControllerBasedStatusBarAppearance`) set to `YES` in `Info.plist`. Read about [Configuring iOS](../ios/configuration) for help.
 
+The status bar visibility defaults to visible and the style defaults to `StatusBarStyle.Dark`. You can change these defaults by modifying the `UIStatusBarHidden` and `UIStatusBarStyle` in the `Info.plist`.
```

Comment:
Default value is `StatusBarStyle.Light` as it sets the status bar to `.default` value, which is the one set as default on `CAPBridgeViewController.swift`.

Also, as `UIStatusBarHidden` and `UIStatusBarStyle` are not in the info.plist in the base project, I think it's better to say "by adding" than "by modifying"

Reply:
👍